projects/nlp/data/network_configs/imbu.json

```diff
@@ -0,0 +1,22 @@
+{
+  "sensorRegionConfig": {
+    "regionEnabled": true,
+    "regionName": "sensor",
+    "regionType": "py.LanguageSensor",
+    "regionParams": {
+      "verbosity": 0,
+      "numCategories": 1
+      },
+    "encoders": {}
+  },
+  "classifierRegionConfig": {
+    "regionEnabled": true,
+    "regionName": "classifier",
+    "regionType": "py.KNNClassifierRegion",
+    "regionParams": {
+      "k": 2000,
```

Comment:
We want `k` to always be larger than the number of sequences because for Imbu the inference results should contain all the sequences.